src/core/edit/edit.cell.view.js

```diff
@@ -201,31 +206,58 @@ export class EditCellView extends View {
 				}
 			}),
 			exit: new Command({
-				execute: (cell, e) => {
+				priority: 1,
+				execute: (cell, e, timeout) => {
 					Log.info('cell.edit', 'reset');
 					if (e) {
 						e.stopImmediatePropagation();
 					}
 
 					cell = cell || this.editor.cell;
 					if (cell) {
-						if (this.commit.canExecute(cell)) {
+						if (this.commit.canExecute(cell, e, timeout)) {
 							const originValue = cell.value;
 							const editValue = this.value;
 							if (originValue !== editValue) {
-								this.commit.execute(cell);
+								this.commit.execute(cell, e, timeout);
+								return true;
 							}
 						}
 
-						if (this.cancel.canExecute(cell)) {
-							this.cancel.execute(cell);
+						if (this.cancel.canExecute(cell, e, timeout)) {
+							this.cancel.execute(cell, e, timeout);
 							return true;
 						}
 					}
 
 					return false;
 				}
-			})
+			}),
+			clear: new Command({
+				priority: 1,
+				canExecute: cell => {
+					cell = cell || this.editor.cell;
+					return cell
```

Comment:
Into the continue of theme above
actually ```execute``` returns boolean here, so maybe d.ts file need foor update just